### DISCUSSION TITLE
Is this timelapse plugin active yet?

### DISCUSSION THREAD

**Discussion by NateSchmitz** [Arksine/moonraker#163] [Feature Requests]
https://github.com/FrYakaTKoP/moonraker/blob/dev-timelapse/docs/configuration.md

Down in the config there is a section for the time-lapse plugin but it does not work when enabled in my moonraker.cfg

**Comment by Arksine** [comment]
It isn't a part of the official repo,  I don't know when or if the developer will create a pull request for it.   I have plans to implement a 3rd party plugin API, this might be a good candidate for it.
